.gitignore

```diff
@@ -17,3 +17,5 @@ test-blockchain/start.sh
 contractAddresses.json
 
 tenderly.yaml
+
+contracts/ChainConstants.sol
```

Comment:
We can probably remove this.

Reply:
I suspect it is a generated file (it was newly created in the repo) that's why I ignored it to avoid committing. Any reasons to not do it?